controllers/clusterpullsecret_controller.go

```diff
@@ -59,6 +75,20 @@ func (r *ClusterPullSecretReconciler) Reconcile(req ctrl.Request) (ctrl.Result,
 			}
 		}
 
+		//add secret to service accounts
+		serviceaccounts := &corev1.ServiceAccountList{}
+		r.Client.List(ctx, serviceaccounts)
+		r.Log.Info(fmt.Sprintf("Found %d service accounts", len(serviceaccounts.Items)))
+
+		for _, serviceaccount := range serviceaccounts.Items {
+			serviceaccountName := serviceaccount.Name
+			namespaceName := serviceaccount.Namespace
+			err := r.SecretReconciler.AppendToServiceAccount(pullSecret, namespaceName, serviceaccountName)
```

Comment:
Why can't this be done through the existing call to `r.SecretReconciler.Reconcile(pullSecret, namespaceName)`?